Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWindowsInformationProtectionPolicyWindows10MdmEnrolled/MSFT_IntuneWindowsInformationProtectionPolicyWindows10MdmEnrolled.schema.mof

```diff
@@ -1,3 +1,13 @@
+[ClassVersion("1.0.0.0")]
+class MSFT_DeviceManagementConfigurationPolicyAssignments
```

Comment:
Since it wasn't there before can we instead change it to IntuneWindowsInformationProtectionPolicyWindows10MdmEnrolledAssignments?

Reply:
I generally don't create a new cim instance for assignments because it's always the same and used everywhere in Intune - we already have a lot of resources using that class

Reply:
And that's the problem, whenever you need to adjust something in the CIM instance you need to change ALL resources at the same time just like I had to do to accommodate the *groupDisplayName* property.

As Nik said before for new resources, or in this case an existing one but that didn't have the property, the assignments property should be called the name of the resource with Assignments at the end. Please check how this was done for the IntuneAccountProtection* policies in https://github.com/microsoft/Microsoft365DSC/commit/6576c924ec286f805e8bb19cbcb9f7b22b87c771

Reply:
Fair enough, I wasn't aware of this - changing now